packages/fast-usdc-deploy/test/fast-usdc.test.ts

```diff
@@ -481,16 +591,24 @@ test.serial('makes usdc advance', async t => {
   t.deepEqual(actual, { incarnationNumber: 4 });
 
   const { runInbound } = t.context.bridgeUtils;
+  // simulate acknowledgement of outgoing forward transfer
   await runInbound(
     BridgeId.VTRANSFER,
     buildVTransferEvent({
       sender: poolAccount,
       target: poolAccount,
       sourceChannel: agoricToNobleChannel,
-      sequence: '1',
+      sequence: '3', // XXX brittle, abstract from tests
```

Comment:
`getIndexByEUD` was pretty helpful in `forwarding.test.ts`. WDYT about using it more widely?
```ts
t.is(getIndexByEUD(aliceEud), -2, "second to last tx is alice's");
await transmitVTransferEvent('acknowledgementPacket', -2);
```

Reply:
I'd like to use something like that but those contract tests have a different setup than the deploy tests. I have a separate branch to improve the DX here, which I hope to have time for. Out of scope for this PR tho